client/coral-admin/src/AppRouter.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import {Router, Route, IndexRoute, browserHistory} from 'react-router';
 
-import ModerationQueue from 'containers/ModerationQueue';
-import CommentStream from 'containers/CommentStream';
-import Configure from 'containers/Configure';
+import ModerationQueue from 'containers/ModerationQueue/ModerationQueue';
```

Comment:
Please update filenames to follow the convention that all containers end with "...Container.js".